packages/ast/template-tag/tests/update-javascript/template-only-4.test.ts

```diff
@@ -0,0 +1,170 @@
+import { assert, test } from '@codemod-utils/tests';
+
+import { updateJavaScript } from '../../src/index.js';
+import { data, renameGetters } from '../helpers/update-javascript.js';
+
+// TODO: Fix bug
+test('update-javascript > template-only (4)', function () {
+  const oldFile = [
+    `import type { TOC } from '@ember/component/template-only';`,
+    `import { hash } from '@ember/helper';`,
+    `import { LinkTo } from '@ember/routing';`,
+    `import { ContainerQuery, width } from 'ember-container-query';`,
+    `import { t } from 'ember-intl';`,
+    ``,
+    `import type { Product } from '../../../utils/routes/products';`,
+    `import styles from './card.css';`,
+    `import ProductsProductImage from './image';`,
+    ``,
+    `export function formatPrice(price: number): string {`,
+    `  return \`$\${price}\`;`,
+    `}`,
+    ``,
+    `interface ProductsProductCardSignature {`,
+    `  Args: {`,
+    `    product: Product;`,
+    `    redirectTo?: string;`,
+    `  };`,
+    `}`,
+    ``,
+    `const ProductsProductCardComponent: TOC<ProductsProductCardSignature> =`,
+    `  <template>`,
```

Comment:
Since `<template>` occurs on a new line, adding a marker comment before results in code that is not valid JavaScript.